.github/workflows/maven.yml

```diff
@@ -0,0 +1,41 @@
+name: Java CI with Maven
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+
+    - name: Cache local Maven repository
+      uses: actions/cache@v2
+      with:
+        path: ~/.m2/repository
+        key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          ${{ runner.os }}-maven-
```

Comment:
The `restore-keys` needs to be more specific, such as 
`restore-keys: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}`

Reply:
Why? I am getting cache hits.

```
Cache restored from key: Linux-maven-bbec10d07a76120fa9e23c88830854907d3e0db14291a1b11a2abec4134f35de
```